css/src/atomics/colors.scss

```diff
@@ -138,12 +129,12 @@
 
 	&:nth-of-type(even) {
 		outline-color: $text;
-		background-color: $body-background-medium;
+		background-color: $body-background;
 	}
 
 	&:nth-of-type(odd) {
 		outline-color: $text;
-		background-color: $body-background-dark;
+		background-color: $alternate-background;
```

Comment:
Should we use `$body-background-medium;` here?

Currently `background-color-alternating-grey` looks like this:

![image](https://user-images.githubusercontent.com/64044661/115093511-6f709480-9ecf-11eb-8775-47152e017781.png)

while `background-color-alternating-grey-reverse` looks like this:
![image](https://user-images.githubusercontent.com/64044661/115093713-12c1a980-9ed0-11eb-82c9-17832f702b39.png)

Reply:
Good save! Updated.